engine/graphics/src/opengl/graphics_opengl_private.h

```diff
@@ -110,16 +110,17 @@ namespace dmGraphics
         uint32_t    m_ModificationVersion;
 
     };
-    // TODO: Why this one here!? Not used?
-    struct VertexBuffer
+
+    struct OpenglVertexAttribute
```

Comment:
These are prefixed with OpenGL now so linking doesn't get screwed up if both vulkan and opengl are linked. I had some issues with this from constructors. We should probably rename all types to this format at some point